controller_joint_names:
    - X
    - Y
    - Z
    - A
    - B
    - C
    - vel_variable
